docs/RFCS/sql_privileges.md

```diff
@@ -0,0 +1,197 @@
+- Feature Name: SQL privileges
+- Status: draft
+- Start Date: 2015-08-10
+- RFC PR: [#2054](https://github.com/cockroachdb/cockroach/pull/2054)
+- Cockroach Issue: [#2005](https://github.com/cockroachdb/cockroach/issues/2005)
+
+# Summary
+
+Introduce SQL privileges for databases and tables. The aim is to provide a privilege
+framework familiar to users of popular SQL servers while embracing simplicity.
+
```

Comment:
Maybe add here that the implementation attaches permissions to the descriptors (table, database) as opposed to a central table somewhere, which means their distribution piggybacks on #1743.